### PR TITLE
for review

### PR DESCRIPTION
**Pull request by agyen**
help review pollstar

### CODE REVIEW

File: src/app/response/response.service.ts [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-@Injectable({
-  providedIn: 'root'
-})
-export class ResponseService {
-
-  constructor(private http: HttpClient) { }
-  private responseLink = 'https://options-web.herokuapp.com/api/v1/responses';
-  // private responseLink ='http://localhost:8080/api/v1/responses'
-
-  recordResponse(result): Observable<Response>{
```
Comment (YaaNuamah) [inline src/app/response/response.service.ts]: The naming of your components should be straightforward

File: src/app/appservice.service.spec.ts [hunk 2]
```diff
@@ -1,12 +0,0 @@
-import { TestBed } from '@angular/core/testing';
-
```
Comment (FrancisBilla) [inline src/app/appservice.service.spec.ts]: kindly remove whitespaces

File: src/app/appservice.service.ts [hunk 3]
```diff
@@ -1,49 +0,0 @@
...
-  }
-
-  getOptionById(id: string): Observable<Poll>{
-    return this.http.get<Poll>(this._option_url + id);
-  }
-
-  addOptions(option:Options): Observable<Options>{
-    // option.forEach(e =>)
-    return this.http.post<Options>(this._option_url, option);
-
```
Comment (FrancisBilla) [inline src/app/appservice.service.ts]: remove whitespace

File: src/app/footer/footer.component.ts [hunk 4]
```diff
@@ -1,15 +0,0 @@
...
-  templateUrl: './footer.component.html',
-  styleUrls: ['./footer.component.scss']
-})
-export class FooterComponent implements OnInit {
-  window = window;
-  constructor() { }
-
-  ngOnInit() {
-  }
-
```
Comment (FrancisBilla) [inline src/app/footer/footer.component.ts]: remove white space

File: src/app/header/header.component.html [hunk 5]
```diff
@@ -1,18 +0,0 @@
...
-      <mat-menu #appMenu="matMenu">
-
-        <button mat-menu-item (click)="manage()" aria-label="Manage polls">
-            <mat-icon>favorite_border</mat-icon>
-            <span>My polls</span>
-          </button>
-
-    </mat-menu>
-    <button mat-icon-button [matMenuTriggerFor]="appMenu" aria-label="Menu">
-    <mat-icon>menu</mat-icon>
```
Comment (FrancisBilla) [inline src/app/header/header.component.html]: kindly indent it

File: src/app/header/header.component.html [hunk 6]
```diff
@@ -1,18 +0,0 @@
...
-        <mat-icon>add</mat-icon>
-    </button>
-      <span class="application-name" (click)="goToRoot()">Pollster</span>
-      
-      <mat-menu #appMenu="matMenu">
-
-        <button mat-menu-item (click)="manage()" aria-label="Manage polls">
-            <mat-icon>favorite_border</mat-icon>
-            <span>My polls</span>
-          </button>
```
Comment (FrancisBilla) [inline src/app/header/header.component.html]: check indent

File: src/app/matpollform/matpollform.component.html [hunk 7]
```diff
@@ -1,37 +0,0 @@
-<mat-form-field class="full-width" >
-    <input matInput placeholder="Question of the poll"  [(ngModel)]="poll.question">
-       <!-- (keyup.enter)="update(poll.name)"  (blur)="update(poll.name)"> -->
-  </mat-form-field>
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.html]: check indentation

File: src/app/matpollform/matpollform.component.html [hunk 8]
```diff
@@ -1,37 +0,0 @@
...
-
-
-  <mat-card class="option-card"  *ngFor="let option of poll.options; let i = index">
-    <mat-form-field class="full-width">
-      <input matInput placeholder="Option {{ i + 1 }}" [(ngModel)]="option.content"  (keyup.enter)="addOption()">
-
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.html]: remove white space

File: src/app/matpollform/matpollform.component.html [hunk 9]
```diff
@@ -1,37 +0,0 @@
...
-      <input matInput placeholder="Option {{ i + 1 }}" [(ngModel)]="option.content"  (keyup.enter)="addOption()">
-      
-    </mat-form-field>
-    <mat-icon class="close" (click)="remove(option)" style="cursor: pointer;">close</mat-icon>
-  </mat-card>
-
-<br>
-  <div style="padding-top: 30px;">
-    <button mat-raised-button (click)="addOption()" aria-label="Add option">Add option</button>
-    <!-- <button mat-raised-button color="primary" class="save-btn" (click)="save()" aria-label="Save">Save</button> -->
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.html]: do away with comments, trust your code :(

File: src/app/matpollform/matpollform.component.ts [hunk 10]
```diff
@@ -1,119 +0,0 @@
...
-     
-    this.poll = {
-      recipient_email: '',
-      poll_id: this.pollid,
-      question:'',
-      options: [],
-      creator_email: 'yaa@turntabl.io',
-      selectMultiple: false,
-    };
-
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: remove white space

File: src/app/matpollform/matpollform.component.ts [hunk 11]
```diff
@@ -1,119 +0,0 @@
...
-
-  }
-
-
-  ngOnInit() {
-  }
-
-  onSubmit(){
-
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: white space

File: src/app/matpollform/matpollform.component.ts [hunk 12]
```diff
@@ -1,119 +0,0 @@
...
-
-  ngOnInit() {
-  }
-
-  onSubmit(){
-    
-    this.user.email = this.creatorsForm.value.email;
-    console.log(this.user.email);
-
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: remove white space

File: src/app/matpollform/matpollform.component.ts [hunk 13]
```diff
@@ -1,119 +0,0 @@
...
-  ngOnInit() {
-  }
-
-  onSubmit(){
-    
-    this.user.email = this.creatorsForm.value.email;
-    console.log(this.user.email);
-  
-    // this.sendmailService.sendmail(this.creatorsForm.value).subscribe()
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: remove comments

File: src/app/matpollform/matpollform.component.ts [hunk 14]
```diff
@@ -1,119 +0,0 @@
...
-  
-    // console.log(this.creatorsForm.value);
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: remove comment

File: src/app/matpollform/matpollform.component.ts [hunk 15]
```diff
@@ -1,119 +0,0 @@
...
-    const index: number = this.poll.options.findIndex(x => x.option_id === id);
-    this.poll.options.splice(index, 1);
-  }
-
-
-
-  send() {
-    this.appservice.addNewPoll(this.poll)
-    .subscribe(response => {
-        console.log(response);
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: do not console log

File: src/app/matpollform/matpollform.component.ts [hunk 16]
```diff
@@ -1,119 +0,0 @@
...
-    });
-
-
-
-// setTimeout(() => {
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: remove comment

File: src/app/matpollform/matpollform.component.ts [hunk 17]
```diff
@@ -1,119 +0,0 @@
...
-//   this.poll.options.forEach(e => {
-//     this.appservice.addOptions(e).subscribe(response => {
-//       console.log(response);
-      
-//     })
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: remove comment

File: src/app/matpollform/matpollform.component.ts [hunk 18]
```diff
@@ -1,119 +0,0 @@
...
-//   })
-// }, 2000);
-
-console.log(this.poll);
```
Comment (FrancisBilla) [inline src/app/matpollform/matpollform.component.ts]: avoid console log

File: src/app/model/model.component.ts [hunk 19]
```diff
@@ -1,15 +0,0 @@
...
-  templateUrl: './model.component.html',
-  styleUrls: ['./model.component.scss']
-})
-export class ModelComponent implements OnInit {
-
-  constructor() { }
-
-  ngOnInit() {
-  }
-
```
Comment (FrancisBilla) [inline src/app/model/model.component.ts]: remove whitespace

File: src/app/thankyoupage/thankyoupage.component.ts [hunk 20]
```diff
@@ -1,38 +0,0 @@
...
-      poll_id: ''
-    };
-
-     ngOnInit(){
-  setTimeout(() => {
-    this.activatedRoute.queryParams.
-    subscribe(params => {
-     this.responses.option_id = params.option_id
-     this.responses.poll_id = params.poll_id
-     // this.responses.suggestions = params.suggestions        // this.responses = params
```
Comment (FrancisBilla) [inline src/app/thankyoupage/thankyoupage.component.ts]: remove comment

File: src/app/thankyoupage/thankyoupage.component.ts [hunk 21]
```diff
@@ -1,38 +0,0 @@
...
-    
-      this.responseService.recordResponse(this.responses).subscribe(response => {
-       console.log(response);
```
Comment (FrancisBilla) [inline src/app/thankyoupage/thankyoupage.component.ts]: avoid console log

File: src/app/thankyoupage/thankyoupage.component.ts [hunk 22]
```diff
@@ -1,38 +0,0 @@
...
-     this.responses.poll_id = params.poll_id
-     // this.responses.suggestions = params.suggestions        // this.responses = params
-    
-      this.responseService.recordResponse(this.responses).subscribe(response => {
-       console.log(response);       
-     })
-   
-   }) 
-  }, 1000);
-
```
Comment (FrancisBilla) [inline src/app/thankyoupage/thankyoupage.component.ts]: remove white space

**Pull request closed without merging**